### DISCUSSION TITLE
I keep getting denied from the education program because of my email

### DISCUSSION THREAD

**Discussion by StenSmeets1** [community/community#62106] [GitHub Education]
### Hi everyone,

Hello all, 

I have tried to apply to the education program 5 times now and each time I get denied for the reason "Attendees of this school must verify and use their academic email address.". The problem with this is that my school doesnt give out school emails. Anyone got advice on how I can apply otherwise? I have send in my proof of enrollment but thats not enough apparantly.

**Comment by pinkyplant** [comment]
Same here

**Comment by anuragdaksh7** [comment]
Same here

**Comment by pinkyplant** [comment]
Gonna post this on twitter to get github's attention.

**Comment by SandyHoffmann** [comment]
Same

**Comment by nonvenomous** [comment]
Same issue here.
Here a bigger thread filled with people ignored by the github support https://github.com/orgs/community/discussions/59360
